boards.txt

```diff
@@ -12717,6 +12717,218 @@ dfrobot_romeo_esp32s3.menu.EraseFlash.none.upload.erase_cmd=
+dfrobot_lorawan_esp32s3.upload.flags=
+dfrobot_lorawan_esp32s3.upload.extra_flags=
+dfrobot_lorawan_esp32s3.upload.use_1200bps_touch=false
+dfrobot_lorawan_esp32s3.upload.wait_for_upload_port=false
+
+dfrobot_lorawan_esp32s3.serial.disableDTR=false
+dfrobot_lorawan_esp32s3.serial.disableRTS=false
+
+dfrobot_lorawan_esp32s3.build.tarch=xtensa
+dfrobot_lorawan_esp32s3.build.bootloader_addr=0x0
+dfrobot_lorawan_esp32s3.build.target=esp32s3
+dfrobot_lorawan_esp32s3.build.mcu=esp32s3
+dfrobot_lorawan_esp32s3.build.core=esp32
+dfrobot_lorawan_esp32s3.build.variant=dfrobot_lorawan_esp32s3
+dfrobot_lorawan_esp32s3.build.board=DFROBOT_LORAWAN_ESP32S3
+dfrobot_lorawan_esp32s3.build.usb_mode=1
+dfrobot_lorawan_esp32s3.build.cdc_on_boot=1
+dfrobot_lorawan_esp32s3.build.msc_on_boot=0
+dfrobot_lorawan_esp32s3.build.dfu_on_boot=0
+dfrobot_lorawan_esp32s3.build.f_cpu=240000000L
+dfrobot_lorawan_esp32s3.build.flash_size=4MB
+dfrobot_lorawan_esp32s3.build.flash_freq=80m
+dfrobot_lorawan_esp32s3.build.flash_mode=dio
+dfrobot_lorawan_esp32s3.build.boot=qio
+dfrobot_lorawan_esp32s3.build.boot_freq=80m
+dfrobot_lorawan_esp32s3.build.partitions=default
+dfrobot_lorawan_esp32s3.build.defines=-D{build.band}
+dfrobot_lorawan_esp32s3.build.loop_core=
+dfrobot_lorawan_esp32s3.build.event_core=
+dfrobot_lorawan_esp32s3.build.psram_type=qspi
+dfrobot_lorawan_esp32s3.build.memory_type={build.flash_type}_{build.psram_type}
+dfrobot_lorawan_esp32s3.build.region=
+
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio=QIO 80MHz
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio.build.flash_mode=dio
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio.build.boot=qio
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio.build.boot_freq=80m
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio.build.flash_freq=80m
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio.build.flash_type=qio
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio120=QIO 120MHz
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio120.build.flash_mode=dio
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio120.build.boot=qio
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio120.build.boot_freq=120m
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio120.build.flash_freq=80m
+dfrobot_lorawan_esp32s3.menu.FlashMode.qio120.build.flash_type=qio
+dfrobot_lorawan_esp32s3.menu.FlashMode.dio=DIO 80MHz
+dfrobot_lorawan_esp32s3.menu.FlashMode.dio.build.flash_mode=dio
+dfrobot_lorawan_esp32s3.menu.FlashMode.dio.build.boot=dio
+dfrobot_lorawan_esp32s3.menu.FlashMode.dio.build.boot_freq=80m
+dfrobot_lorawan_esp32s3.menu.FlashMode.dio.build.flash_freq=80m
+dfrobot_lorawan_esp32s3.menu.FlashMode.dio.build.flash_type=qio
+dfrobot_lorawan_esp32s3.menu.FlashMode.opi=OPI 80MHz
+dfrobot_lorawan_esp32s3.menu.FlashMode.opi.build.flash_mode=dout
+dfrobot_lorawan_esp32s3.menu.FlashMode.opi.build.boot=opi
+dfrobot_lorawan_esp32s3.menu.FlashMode.opi.build.boot_freq=80m
+dfrobot_lorawan_esp32s3.menu.FlashMode.opi.build.flash_freq=80m
+dfrobot_lorawan_esp32s3.menu.FlashMode.opi.build.flash_type=opi
```

Comment:
I think this option should be removed, if the module you use does not have OPI Flash. Please check
```suggestion
```